docs/test/test.md

```diff
@@ -56,3 +56,55 @@ cd test/bdd/fixtures/agent-wasm
 
 - To access user agent wasm open [user home page](https://127.0.0.1:8091/dashboard).
 - To access second user agent wasm open [user home page](https://127.0.0.1:8071/dashboard).
+
+## How to establish a did-connection between agents?
+
+1. Go to `Connections` page and register the router for both agents. Use `http://localhost:10093` as a router URL.
```

Comment:
Don't we need a connection between user-agent and router before registering ?

Reply:
I am creating it under the hood.